packages/cf-layout/src/organisms/featured-content-module.less

```diff
@@ -19,25 +19,31 @@
         padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
         padding-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
 
+        .respond-to-max( @bp-xs-max, {
+            padding-right: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+            padding-left: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+        } );
+
         .respond-to-min( @fcm-bp-min, {
-            padding-right: unit( @grid_gutter-width / @base-font-size-px, em );
+            flex: 1;
+            padding-right: 300px;
             padding-left: unit( @grid_gutter-width / @base-font-size-px, em );
-            width: 300px; // Width of 5 of 12 cols at lg size
-            float: left;
         } );
     }
 
     &_visual {
+        .respond-to-max( @bp-xs-max, {
+            padding-right: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+            padding-bottom: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
+            padding-left: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
```

Comment:
Three lines with extra parens. I'll leave it up to you if you want to remove them. We have both formats in the wild I think.